packages/launchpad/src/setup/FrameworkOptionsFooter.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="flex bg-gray-50 h-65px py-12px px-15px text-gray-800 items-center">
+    <div><IconComponent class="h-40px min-w-0 min-h-0 w-40px" /></div>
+    <div class="ml-10px">
+      <div>{{ t("setupPage.projectSetup.frameworkNotListed") }}</div>
+      <div>
+        <ExternalLink
+          href="https://on.cypress.io/component-integrations"
+          class="text-indigo-500 block hocus-link-default group"
+        >
+          {{ t('setupPage.projectSetup.browseIntegrations') }}<i-cy-arrow-right_x16
+            class="ml-4px transform transition-transform ease-in -translate-y-1px duration-200 inline-block icon-dark-current group-hocus:translate-x-2px"
+          />
+        </ExternalLink>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { useI18n } from '@cy/i18n'
+import ExternalLink from '@packages/frontend-shared/src/gql-components/ExternalLink.vue'
+import IconComponent from '~icons/cy/testing-type-component_x64.svg'
```

Comment:
Left a comment on this is Slack also, but we should try using this icon and the component from the design system:
https://design.cypress.io/?path=/story/react_icon--icon&args=name:general-grid-2x2

Reply:
Done https://github.com/cypress-io/cypress/pull/25831/commits/b8f643a7cd176e754e0e328f267852ff40333117